content/sensu-go/5.13/reference/rbac.md

```diff
@@ -1179,3 +1224,5 @@ You can add these resources to Sensu using [`sensuctl create`][31].
 [35]: #cluster-role-bindings
 [36]: ../../sensuctl/reference#creating-resources-across-namespaces
 [37]: ../license
+[38]: ../../installation/auth/#groups-prefix
```

Comment:
This links to the `groups_prefix` in the LDAP spec (https://docs.sensu.io/sensu-go/5.14/installation/auth/#spec-attributes)

Since this example uses AD, I think we would need to add something like `<a name="ad-groups-prefix"></a>` just before the AD `groups-prefix` description in https://docs.sensu.io/sensu-go/5.14/installation/auth/#active-directory-spec-attributes, right?

I guess we could also revise the role/role binding with groups prefix examples section to use LDAP rather than AD, but I assume there is a good reason you used AD there. What do you think?

Reply:
There seems to already be a `<a name="ad-groups-prefix"></a>` before the groups prefix. So https://docs.sensu.io/sensu-go/5.14/installation/auth/#ad-groups-prefix works already!

I'll make the change so we use AD everywhere.